packages/@expo/use-updates/README.md

```diff
@@ -0,0 +1,64 @@
+# @expo/use-updates
+
+A new, simpler JS API for the `expo-updates` module.
```

Comment:
```suggestion
A simpler JavaScript API for the `expo-updates` module.
```

Let's skip "new" in the description. If we want to indicate the package freshness in some way, I would go with information note at the top (similar to `expo-maps` for example).